db_stress_tool/db_stress_listener.cc

```diff
@@ -37,10 +38,29 @@ UniqueIdVerifier::UniqueIdVerifier(const std::string& db_name, Env* env)
     exit(1);
   }
 
+  std::string tmp_path = path_ + ".tmp";
```

Comment:
Suggest a comment like `// Avoid relying on ReopenWritableFile which is not available everywhere`